lib/readline.js

```diff
@@ -728,10 +732,9 @@ Interface.prototype._getDisplayPos = function(str) {
       offset = 0;
       continue;
     }
-    // Tabs must be aligned by an offset of 8.
-    // TODO(BridgeAR): Make the tab size configurable.
+    // Tabs must be aligned by an offset of the tab size.
     if (char === '\t') {
-      offset += 8 - (offset % 8);
+      offset += this.tabSize - (offset % this.tabSize);
```

Comment:
```suggestion
      if (this.tabSize > 0)
        offset += this.tabSize - (offset % this.tabSize);
```
Otherwise `offset` turns into a nan when `this.tabSize === 0`. Can you add a regression test for that case?

Reply:
I changed the minimum to 1 instead. That seemed more logical as lower bound.